elide-core/src/main/java/com/yahoo/elide/security/executors/ActivePermissionExecutor.java

```diff
@@ -430,9 +435,11 @@ private ExpressionResult executeExpressions(final Expression expression,
             return DEFERRED;
         }
         if (result == FAIL) {
-            ForbiddenAccessException e = new ForbiddenAccessException(annotationClass.getSimpleName(),
+            ForbiddenAccessException e = new ForbiddenAccessException(getSimpleName(annotationClass),
```

Comment:
ditto above

Reply:
`import static com.yahoo.elide.core.EntityDictionary.getSimpleName;`

to make it less cluttered.  Is it more readable spelling it out?

Reply:
I think it's a bit more readable to spell out.

Reply:
Done